src/policy/policy.h

```diff
@@ -75,16 +75,23 @@ CAmount GetDustThreshold(const CTxOut& txout, const CFeeRate& dustRelayFee);
 bool IsDust(const CTxOut& txout, const CFeeRate& dustRelayFee);
 
 bool IsStandard(const CScript& scriptPubKey, TxoutType& whichType);
-    /**
-     * Check for standard transaction types
-     * @return True if all outputs (scriptPubKeys) use only standard transaction forms
-     */
+
+
+// Changing the default transaction version requires a two step process: first
+// adapting relay policy by bumping TX_MAX_STANDARD_VERSION, and then later
+// allowing the new transaction version in the wallet/RPC.
+static constexpr decltype(CTransaction::nVersion) TX_MAX_STANDARD_VERSION{3};
```

Comment:
20611: nit: could probably use `CTransaction:SPECIAL_VERSION` here
```suggestion
static constexpr decltype(CTransaction::nVersion) TX_MAX_STANDARD_VERSION{CTransaction::SPECIAL_VERSION};
```

Reply:
that's not dash code but backported, I'd prefer to keep as it is. 

They do not re-use definitions:
```
static const int32_t CURRENT_VERSION=2;
static constexpr decltype(CTransaction::nVersion) TX_MAX_STANDARD_VERSION{2};

```